newsroom/push/publishing.py

```diff
@@ -271,6 +276,9 @@ async def publish_planning_into_event(self, planning: dict[str, Any]) -> Optiona
 
         agenda = deepcopy(orig_agenda)
 
+        if agenda.get("item_type") == "event" and planning["_id"] not in agenda["planning_ids"]:
+            agenda["planning_ids"].append(planning["_id"])
+
         if (
```

Comment:
We do not want to process any of these other conditions for events with a secondary link. These are side affects we want to avoid. If the link is secondary, then we only want to add to the `planning_ids` and `coverages`, and nothing else. In this case may I suggest a different function altogether if we're processing a secondary link